### ISSUE TITLE
Top bar styling looks awful in chrome

### ISSUE THREAD

**Issue by tgetgood** [openmindanesth/openmind#31]
So far I've only been testing in firefox. 

We need to start testing on all the major browsers.

**Comment by hlesaint** [comment]
Good point. For Brave everything looks good. Older browsers could be recognized and displayed: not optimized for your browser, layout and function could be compromised.
